frontend/src/libs/components/checkbox/common.module.scss

```diff
@@ -0,0 +1,67 @@
+@use "sass:color";
+
+@import "src/assets/css/styles.scss";
+
+.container {
+  display: flex;
+  flex-direction: column;
+  font-family: $font-family;
+}
+
+.error {
+  color: $red-dark;
+  font-weight: $font-weight-regular;
+  font-size: 12px;
+  line-height: calc(15.12 / 12);
+}
+
+.label:global(.disabled) + .error {
+  color: color.adjust(
+    $red-dark,
+    $lightness: 35%
+  ); // Disabled color for error messages
+}
+
+.label {
+  --color: #{$blue-dark};
+  --color-text: #{$black};
+
+  display: flex;
+  align-items: baseline;
+  gap: 8px;
+  width: fit-content;
+  user-select: none;
+  min-height: calc(
+    16px * $line-height-xl
+  ); // Left this instead of line-height 140% to normalize height of the element without label
+
+  font-weight: $font-weight-semibold;
+  font-style: normal;
+}
+
+.labelText {
+  color: var(--color-text);
+  transition: $anim-normal color $anim-timing-fn-ui;
+}
+
+.label:hover {
+  --color: #{$blue-light};
+  --color-text: #{color.adjust($black, $lightness: 20%)};
```

Comment:
why not simply color + variable from vars file?

Reply:
because this is more versatile. You can have a single color variable in vars file, while this:
color.adjust($black, $lightness: 20%)}
is just a modifier of the color. There is no point of having like 5 variations of a single color for every need, when we have such powerful tools like color.adjust and others.

Why wouldn't we use them?